components/analytics-tools/org.wso2.carbon.analytics.tools.backup/src/main/java/org/wso2/carbon/analytics/tools/backup/AnalyticsDataBackupTool.java

```diff
@@ -578,6 +587,29 @@ private static void createDirectoryInLocalSystem(String path) {
     }
 
     /**
+     * Returns an indexing disabled schema from the provided Analytics Schema.
```

Comment:
Explain the parameter in the java docs, have the space between the method comment to parameters' comments.